includes/class-wc-calypso-bridge-setup.php

```diff
@@ -224,7 +224,7 @@ public function woocommerce_create_pages_callback() {
 			$this->write_to_log( $operation, 'INITIALIZED' );
 
 			// Set the operation as completed if the store is active for more than 10 minutes.
```

Comment:
Nit: the comment says 10 minutes, but the code is now set to 5 minutes.
```suggestion
			// Set the operation as completed if the store is active for more than 5 minutes.
```